README.md

```diff
@@ -123,6 +171,65 @@ monokle validate --output sarif k8s-dir > results.sarif
 
 Afterward you could use [VSC's SARIF Viewer][vsc-sarif] or other tools to inspect the results.
 
+## Using with Monokle Cloud
+
+To use remote policy with Monokle CLI you will need to create a project and configure policy for it in Monokle Cloud. Start by signing in to [Monokle Cloud](https://app.monokle.com).
+
+> In case of doubts, refer to [Getting Started Guide](https://docs.monokle.com/tutorials/getting-started) or hit us directly on [Discord](https://discord.com/invite/6zupCZFQbe).
+
+### Project setup
+
+After signing up, start by creating a project on [Projects page](https://app.monokle.com/dashboard/projects):
+
+<p align="center">
+  <img src="docs/images/projects.png"/>
+</p>
+
+### Add your repository to your project
+
+After project is created, add a repository (the one you will be working locally with) to a project. This can be done by going to `Repositories` tab in project view and using `Add repository` button:
+
+<p align="center">
+  <img src="docs/images/repos.png"/>
+</p>
+
+### Policy setup
+
+The last step is policy setup. You can use policy wizard by going to `Policy` tab in project view:
+
+<p align="center">
+  <img src="docs/images/policies.png"/>
+</p>
+
+After the setup is done, you can run `monokle validate` command and it will use remote policy as long as you are logged in.
+
+## Using on CI/CD pipelines
+
+> We have dedicated [`Monokle GitHub Bot`](#monokle-github-bot) to integrate centralized policy management into GitHub Actions which gives more extensive integration.
```

Comment:
You might have intended something else here? Below you correctly mention the Monokle GitHub Action in a comment but here you talk about the GH Bot wrt GitHub Actions?

Reply:
Most probably 👍 I guess we should both mention GH Action and GH Bot here, right? GH Bot as being more "integrated" (with the rest of ecosystem) solution?

Reply:
Ahhhh, I know what I was trying to say, I used GitHub Action as a "generic" name describing GitHub CI/CD (because that's how [GH calls it](https://github.com/features/actions)). Meaning that you can use Monokle GH Bot to easily integrated validation into GitHub CI/CD. I'll try to reword it to make easier to understand.